packages/renderer/src/lib/recommendation/ExtensionBanner.spec.ts

```diff
@@ -60,8 +60,24 @@ const imageBackground: IExtensionBanner = {
   },
 };
 
+const showMessageBoxMock = vi.fn();
+const telemetryTrackMock = vi.fn();
+
+beforeAll(() => {
+  Object.defineProperty(window, 'showMessageBox', {
+    value: showMessageBoxMock,
+  });
+  Object.defineProperty(window, 'telemetryTrack', {
+    value: telemetryTrackMock,
+  });
+  Object.defineProperty(window, 'updateConfigurationValue', {
+    value: vi.fn(),
+  });
+});
```

Comment:
following https://github.com/podman-desktop/podman-desktop/pull/10426 you can just forget these defitions